### PR TITLE
Update2 iq.m3u 

### PR DESCRIPTION
**Pull request by Redo-San**
I added more channels and verified that they are working and deleted some closed channels and updated the links of some channels as well

### CODE REVIEW

File: streams/iq.m3u [hunk 1]
```diff
@@ -5,14 +5,34 @@ http://rtmp1.abnsat.com/hls/arabic.m3u8
 https://5ab29cc78f681.streamlock.net/afarinTV/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="AlEtejah.iq",Al-Etejah (576p) [Not 24/7]
 http://streaming.aletejahtv.iq:1935/etejah-newLive/smil:etejahNew.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="AlghadeerTV.iq",AlGhadeer (720p) [Not 24/7]
-https://sd343444.vodu.store:3356/live/Alghadeer/index.m3u8
+#EXTINF:-1 tvg-id="AlforatTV.iq",Alforat TV (1080p) 
+http://95.216.180.111:1935/live/10/playlist.m3u8
+#EXTINF:-1 tvg-id="AlGhadTV.eg",Al Ghad (720p) [Not 24/7]
```
Comment (UltraHDR) [inline streams/iq.m3u:10]: Only iraq channels are added to iq.m3u. Please remove channels with a tvg-id that does not end with `.iq` and add them to the playlist for that country.

File: streams/iq.m3u [hunk 2]
```diff
@@ -5,14 +5,34 @@ http://rtmp1.abnsat.com/hls/arabic.m3u8
...
+#EXTINF:-1 tvg-id="AlJazeeraMubasher.qa",Al Jazeera Mubasher (1080p)
+https://live-hls-web-ajm.getaj.net/AJM/index.m3u8
+#EXTINF:-1 tvg-id="AlMaarefTV.kw",Al Maaref (350p)
+https://5e74a9d684b2e.streamlock.net/liveTrans/ngrp:channel23_all/playlist.m3u8
+#EXTINF:-1 tvg-id="AlMayadeenTV.lb",Al Mayadeen (576p)
+https://mdnlv.cdn.octivid.com/almdn/smil:mpegts.stream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AlNaeemTV.iq",Al-Naeem TV (576p)
+https://nl2.livekadeh.com/hls2/alnaeem_high.m3u8
+#EXTINF:-1 tvg-id="alrabiaa.tv",Al Rabiaa TV (1080p)
+https://cllive.itworkscdn.net/alrabiaatvnewslive/token=nva=1690991331~dirs=1~hash=058a79456eeadb3deb21b/alrabiaatv.smil/playlist.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:35]: Expires

File: streams/iq.m3u [hunk 3]
```diff
@@ -5,14 +5,34 @@ http://rtmp1.abnsat.com/hls/arabic.m3u8
...
+#EXTINF:-1 tvg-id="AlJazeeraDocumentary.qa",Al Jazeera Documentary (1080p) [Geo-blocked]
+https://live-hls-web-ajd.getaj.net/AJD/index.m3u8
+#EXTINF:-1 tvg-id="AlJazeeraMubasher.qa",Al Jazeera Mubasher (1080p)
+https://live-hls-web-ajm.getaj.net/AJM/index.m3u8
+#EXTINF:-1 tvg-id="AlMaarefTV.kw",Al Maaref (350p)
+https://5e74a9d684b2e.streamlock.net/liveTrans/ngrp:channel23_all/playlist.m3u8
+#EXTINF:-1 tvg-id="AlMayadeenTV.lb",Al Mayadeen (576p)
+https://mdnlv.cdn.octivid.com/almdn/smil:mpegts.stream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AlNaeemTV.iq",Al-Naeem TV (576p)
+https://nl2.livekadeh.com/hls2/alnaeem_high.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:33]:
```suggestion
https://nl2.livekadeh.com/hls2/alnaeem.m3u8
```

File: streams/iq.m3u [hunk 4]
```diff
@@ -29,16 +49,46 @@ https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
...
+#EXTINF:-1 tvg-id="Alqanat9TV.iq",Alqanat9 TV (1080p)
+https://cdn.bestream.io:19360/alqanat9/alqanat9.m3u8
 #EXTINF:-1 tvg-id="AmozhgaryTV.iq",Amozhgary TV (1080p)
 https://media.streambrothers.com:1936/8248/8248/playlist.m3u8
 #EXTINF:-1 tvg-id="AssyrianANB.iq",Assyrian ANB (720p)
 https://anbsat.ddns.net/anb/live/ANB/playlist.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-http://91.134.145.75:10001/Dijlah/index.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-https://ghaasiflu.online/Dijlah/index.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:39]: Works

File: streams/iq.m3u [hunk 5]
```diff
@@ -29,16 +49,46 @@ https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
...
 #EXTINF:-1 tvg-id="AlSharqiyaNews.iq",Al-Sharqiya News (1080p)
 https://5d94523502c2d.streamlock.net/alsharqiyalive/mystream/playlist.m3u8
+#EXTINF:-1 tvg-id="Alqanat9TV.iq",Alqanat9 TV (1080p)
+https://cdn.bestream.io:19360/alqanat9/alqanat9.m3u8
 #EXTINF:-1 tvg-id="AmozhgaryTV.iq",Amozhgary TV (1080p)
 https://media.streambrothers.com:1936/8248/8248/playlist.m3u8
 #EXTINF:-1 tvg-id="AssyrianANB.iq",Assyrian ANB (720p)
 https://anbsat.ddns.net/anb/live/ANB/playlist.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-http://91.134.145.75:10001/Dijlah/index.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:37]: Works

File: streams/iq.m3u (hunk 1 above)
Comment (Redo-San) [reply streams/iq.m3u:10]: Ok

File: streams/iq.m3u [hunk 6]
```diff
@@ -29,16 +29,32 @@ https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
...
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-https://ghaasiflu.online/Dijlah/index.m3u8
+#EXTINF:-1 tvg-id="BayyinatTV.iq",Bayyinat TV (404p)
+https://nl2.livekadeh.com/hls2/Bayyinat_high.m3u8
+#EXTINF:-1 tvg-id="BeitolAbbasTVChannel.iq",BeitolAbbas TV Channel (720p)
+https://live.beitolabbas.tv/live/beitolabbastv.m3u8
+#EXTINF:-1 tvg-id="dijlah.tv",Dijlah TV (1080p)
+https://ghaasiflu.online/Dijlah/tracks-v1a1/mono.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:43]:
```suggestion
https://ghaasiflu.online/Dijlah/index.m3u8
```

File: streams/iq.m3u (hunk 5 above)
Comment (UltraHDR) [reply streams/iq.m3u:37]: Please add it back

File: streams/iq.m3u [hunk 7]
```diff
@@ -29,16 +29,32 @@ https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
...
-http://91.134.145.75:10001/Dijlah/index.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
+#EXTINF:-1 tvg-id="BayyinatTV.iq",Bayyinat TV (404p)
+https://nl2.livekadeh.com/hls2/Bayyinat_high.m3u8
+#EXTINF:-1 tvg-id="BeitolAbbasTVChannel.iq",BeitolAbbas TV Channel (720p)
+https://live.beitolabbas.tv/live/beitolabbastv.m3u8
+#EXTINF:-1 tvg-id="dijlah.tv",Dijlah TV (1080p)
 https://ghaasiflu.online/Dijlah/index.m3u8
+#EXTINF:-1 tvg-id="DuaChannel.iq",Dua Channel (720p)
+https://live.ishiacloud.com/haditv.co.uk/dua-channel.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:45]: Not working

File: streams/iq.m3u [hunk 8]
```diff
@@ -65,6 +79,16 @@ https://5a3ed7a72ed4b.streamlock.net/live/SMIL:myStream.smil/playlist.m3u8
 https://ikomg2.mmdlive.lldns.net/ikomg2/366ca61db4c646a2bf9cd9f92dd6321c/manifest.m3u8
 #EXTINF:-1 tvg-id="KurdsatNews.iq",KurdSat News (1080p) [Not 24/7]
 https://ikomg2.mmdlive.lldns.net/ikomg2/107b7df8f5444d778f349100739a09cd/manifest.m3u8
+#EXTINF:-1 tvg-id="Kurdsat.iq",Kurdsat (1080p)
+https://ikomg2.mmdlive.lldns.net/ikomg2/366ca61db4c646a2bf9cd9f92dd6321c/manifest.m3u8
+#EXTINF:-1 tvg-id="KurdsatNews.iq",KurdSat News (1080p) [Not 24/7]
+https://ikomg2.mmdlive.lldns.net/ikomg2/107b7df8f5444d778f349100739a09cd/manifest.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:85]: already there on line 79

File: streams/iq.m3u [hunk 9]
```diff
@@ -3,16 +3,16 @@
 http://rtmp1.abnsat.com/hls/arabic.m3u8
 #EXTINF:-1 tvg-id="AfarinTV.iq",Afarin TV (720p) [Not 24/7]
 https://5ab29cc78f681.streamlock.net/afarinTV/livestream/playlist.m3u8
-#EXTINF:-1 tvg-id="AlEtejah.iq",Al-Etejah (576p) [Not 24/7]
-http://streaming.aletejahtv.iq:1935/etejah-newLive/smil:etejahNew.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="AlghadeerTV.iq",AlGhadeer (720p) [Not 24/7]
-https://sd343444.vodu.store:3356/live/Alghadeer/index.m3u8
+#EXTINF:-1 tvg-id="AlforatTV.iq",Alforat TV (1080p) 
+http://95.216.180.111:1935/live/10/playlist.m3u8
+#EXTINF:-1 tvg-id="AlhurraIraq.us",Alhurra Iraq (720p)
```
Comment (UltraHDR) [inline streams/iq.m3u:8]: Should be in the US.m3u playlist

File: streams/iq.m3u [hunk 10]
```diff
@@ -99,19 +123,19 @@ https://ythls.onrender.com/UCi4CSgJk3i_slWnWG9Y14qA.m3u8
 https://live.ishiacloud.com/haditv.co.uk/dua-channel.m3u8
 #EXTINF:-1 tvg-id="IshtarTV.iq",Ishtar TV (1080p) 
 http://ishtar.cdncast.xyz:1935/live/iShtarHD/playlist.m3u8
+#EXTINF:-1 tvg-id="InewsTV.iq",Inews TV (1080p)
```
Comment (UltraHDR) [inline streams/iq.m3u:126]: Duplicate. already there on line 71

File: streams/iq.m3u (hunk 7 above)
Comment (Redo-San) [reply streams/iq.m3u:45]: But the link works when I run it?

File: streams/iq.m3u (hunk 5 above)
Comment (Redo-San) [reply streams/iq.m3u:37]: > Please add it back

It's an old link and there are always interruptions
The new link I added is much better

File: streams/iq.m3u (hunk 9 above)
Comment (Redo-San) [reply streams/iq.m3u:8]: No, it is an Iraqi channel, speaking in Arabic, but it broadcasts from inside America, so it is not an American channel.

File: streams/iq.m3u (hunk 2 above)
Comment (kaikyoofers) [reply streams/iq.m3u]: @UltraHDR you didnt check there was a incorrect tvg-id.

File: streams/iq.m3u (hunk 9 above)
Comment (UltraHDR) [reply streams/iq.m3u:8]: It's based in the US
https://en.wikipedia.org/wiki/Alhurra

File: streams/iq.m3u [hunk 11]
```diff
@@ -65,53 +83,35 @@ https://5a3ed7a72ed4b.streamlock.net/live/SMIL:myStream.smil/playlist.m3u8
...
-#EXTINF:-1 tvg-id="AlforatTV.iq",Alforat TV (1080p) 
-http://95.216.180.111:1935/live/10/playlist.m3u8
+#EXTINF:-1 tvg-id="WaarTV.iq",Waar TV (480p)
+https://ca-rt.onetv.app/Waar/index-0.m3u8
+#EXTINF:-1 tvg-id="WaarTV.iq",Waar TV (720p)
+https://ikomg4.s.llnwi.net/waarhd/waar-hd.m3u8
+#EXTINF:-1 tvg-id="ZagrosTV.iq",Zagros (720p) [Not 24/7]
+https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="ZagrosTV.iq",Zagros (720p) [Not 24/7]
+https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:117]: Duplicate link

File: streams/iq.m3u (hunk 7 above)
Comment (UltraHDR) [reply streams/iq.m3u:47]: It works in mpv but not vlc so I think it's ok to have it

File: streams/iq.m3u [hunk 12]
```diff
@@ -65,53 +83,35 @@ https://5a3ed7a72ed4b.streamlock.net/live/SMIL:myStream.smil/playlist.m3u8
...
 #EXTINF:-1 tvg-id="RudawTV.iq",Rudaw TV (1080p)
 https://svs.itworkscdn.net/rudawlive/rudawlive.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="WaarTV.iq",Waar TV (480p)
-https://ca-rt.onetv.app/Waar/index-0.m3u8
-#EXTINF:-1 tvg-id="WaarTV.iq",Waar TV (720p)
-https://ikomg4.s.llnwi.net/waarhd/waar-hd.m3u8
-#EXTINF:-1 tvg-id="ZagrosTV.iq",Zagros (720p) [Not 24/7]
-https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="NUBARPlusPlus.iq",NUBAR Plus TV (720p)
-http://stream.nubar.tv:1935/private/NUBARPlus/playlist.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:83]: works

**Review by UltraHDR — DISMISSED** [review]

**Review by UltraHDR — DISMISSED** [review]

**Review by UltraHDR — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]
In case you are wondering : 

http://95.216.180.111:1935/live/1/playlist.m3u8 <- AFAQ TV
http://95.216.180.111:1935/live/2/playlist.m3u8 <- could be Al Ghadeer TV
http://95.216.180.111:1935/live/3/playlist.m3u8 <- Al Noujaba
http://95.216.180.111:1935/live/4/playlist.m3u8 <- Al Sharqiya News
http://95.216.180.111:1935/live/6/playlist.m3u8 <- Al Ayyam TV
http://95.216.180.111:1935/live/7/playlist.m3u8 <- Al Ahd
http://95.216.180.111:1935/live/8/playlist.m3u8 <- Al Arabiya
http://95.216.180.111:1935/live/9/playlist.m3u8 <- Al Hadath
http://95.216.180.111:1935/live/11/playlist.m3u8 <- Alquds Satellite Channel
http://95.216.180.111:1935/live/12/playlist.m3u8 <- Pal Today TV
http://95.216.180.111:1935/live/13/playlist.m3u8 <- Al Anwar 2
http://95.216.180.111:1935/live/14/playlist.m3u8 <- ?
snapshot4

(in case it goes down, screenshot above)

http://95.216.180.111:1935/live/15/playlist.m3u8 <- ifilm Farsi
http://95.216.180.111:1935/live/16/playlist.m3u8 <- Thaqalayn TV
http://95.216.180.111:1935/live/17/playlist.m3u8 <- ?
snapshot5

(in case it goes down, screenshot above)
http://95.216.180.111:1935/live/18/playlist.m3u8 <- Al Janoub
http://95.216.180.111:1935/live/19/playlist.m3u8 <- Al Eshraq TV
http://95.216.180.111:1935/live/20/playlist.m3u8 <- Nabaa TV
http://95.216.180.111:1935/live/21/playlist.m3u8 <- Press TV English
http://95.216.180.111:1935/live/22/playlist.m3u8 <- Al Naeem
http://95.216.180.111:1935/live/23/playlist.m3u8 <- Voice of America Farsi 365
http://95.216.180.111:1935/live/24/playlist.m3u8 <- from Google's own translation : Peace TV
http://95.216.180.111:1935/live/28/playlist.m3u8 <- BBC Persian
http://95.216.180.111:1935/live/29/playlist.m3u8 <- CNN International
http://95.216.180.111:1935/live/30/playlist.m3u8 <- from Google's own translation : Al Ittihad
http://95.216.180.111:1935/live/33/playlist.m3u8 <- Al Etejah TV
Server name : 63b03f7689049.streamlock.net

**Pull request merged**